x-pack/plugins/security_solution/public/management/common/utils.ts

```diff
@@ -28,18 +28,18 @@ const getPolicyQuery = (policyId: string): string => {
   return `exception-list-agnostic.attributes.tags:"policy:${policyId}"`;
 };
 
-export const parsePoliciesToKQL = (includedPolicies: string, excludedPolicies: string): string => {
+export const parsePoliciesToKQL = (
+  includedPolicies: string[],
+  excludedPolicies: string[]
```

Comment:
should we make excluded_policies optional?

Reply:
changes in https://github.com/elastic/kibana/pull/121305/commits/d5ac895d5ad8b5969df21e3387f9345ce31ea65e